ICT4DNews/app/tasks.gradle

```diff
@@ -0,0 +1,21 @@
+configurations {
+    ktlint
+}
+
+task ktlint(type: JavaExec, group: "verification") {
+    description = "Check Kotlin code style."
+    classpath = configurations.ktlint
+    main = "com.github.shyiko.ktlint.Main"
+    args "src/**/*.kt"
```

Comment:
change to `args "src/**/*.kt", "--reporter=plain", "--android"` to target Android